client/src/App.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Home from './components/Home';
```

Comment:
Real minor, but can you also remove the import for the Home component in App.js and delete the Home.js component as well to complete the clean up.  You can just re-commit those last changes to the same branch.

Reply:
No problem - I've made the two changes you requested.